### PR TITLE
[IMP] repair: smarter smart buttons

### PR DESCRIPTION
**Pull request by Sehnde**
Added a todo count field on the button where the lot is the product being repaired. Changed the texts to better fit the buttons.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/repair/models/stock_lot.py [hunk 1]
```diff
@@ -19,9 +20,13 @@ def _compute_repair_order_ids(self):
             lot.repair_order_ids = repair_orders[lot.id]
             lot.repair_order_count = len(lot.repair_order_ids)
 
+    def _compute_in_repair_count(self):
+        for lot in self:
+            lot.in_repair_count = len(self.env['repair.order'].search([('lot_id', '=', lot.id), ('state', 'not in', ('done', 'cancel'))]))
```
Comment (mama-odoo) [inline addons/repair/models/stock_lot.py:25]: Why not use search_count to optimize the query ?
```suggestion
            lot.in_repair_count = self.env['repair.order'].search_count([('lot_id', '=', lot.id), ('state', 'not in', ('done', 'cancel'))]))
```

Comment (Sehnde) [reply addons/repair/models/stock_lot.py]: done

**Comment by amoyaux** [comment]
Even better. Don't do a search like that inside a loop. Use the read_group to perform only one sql query (you can even use a single compute in order to compute both fields at once in a single step)

**Comment by Sehnde** [comment]
back to draft: waiting for additional specs on task 3395161

**Pull request closed without merging**